activerecord/lib/rails/generators/active_record/migration/migration_generator.rb

```diff
@@ -28,7 +28,7 @@ def create_migration_file
         def set_local_assigns!
           @migration_template = "migration.rb"
           case file_name
-          when /^(add|remove)_.*_(?:to|from)_(.*)/
+          when /^(add)_.*_(?:to)_(.*)/, /^(remove)_.*?_(?:to|from)_(.*)/
```

Comment:
thanks for the feedback! I was trying to come up with a table name with "to" on the title but couldn't find it :)